src/lib/index.js

```diff
@@ -1267,3 +1279,28 @@ lib.join2 = function(arr, mainSeparator, lastSeparator) {
 lib.bigFont = function(size) {
     return Math.round(1.2 * size);
 };
+
+/**
+ * Return the mouse position from the last event registered by D3.
+ * @returns An array with two numbers, representing the x and y coordinates of the mouse pointer
+ *   at the event relative to the targeted node.
+ */
+lib.getPositionFromD3Event = function() {
+    var firefoxVersion = lib.getFirefoxVersion();
+
+    // see https://bugzilla.mozilla.org/show_bug.cgi?id=1684973
+    var isProblematicFirefox = firefoxVersion && firefoxVersion < 86;
```

Comment:
What happens in the case of `firefoxVersion == 0` ?
Perhaps we should say
```js
var isProblematicFirefox = firefoxVersion !== null && firefoxVersion < 86;
```

Reply:
fair point, done